dipy/workflows/reconst.py

```diff
@@ -0,0 +1,151 @@
+from dipy.io.gradients import read_bvals_bvecs
+from dipy.reconst.dti import (TensorModel, color_fa, fractional_anisotropy,
+                              geodesic_anisotropy, mean_diffusivity,
+                              axial_diffusivity, radial_diffusivity,
+                              lower_triangular, mode as get_mode)
+from dipy.workflows.workflow import Workflow
+
+
+class ReconstDtiFlow(Workflow):
+    @classmethod
+    def get_short_name(cls):
+        return 'dti'
+
+    def run(self, input_files, bvalues, bvectors, mask_files, b0_threshold=0.0,
+            out_dir='', out_tensor='tensors.nii.gz', out_fa='fa.nii.gz',
+            out_ga='ga.nii.gz', out_rgb='rgb.nii.gz', out_md='md.nii.gz',
+            out_ad='ad.nii.gz', out_rd='rd.nii.gz', out_mode='mode.nii.gz',
+            out_evec='evecs.nii.gz', out_eval='evals.nii.gz'):
+
+        """ Workflow for tensor reconstruction and DTI metrics computing.
+        It a tensor recontruction on the files by 'globing' ``input_files`` and
+        saves the dti metrics in a directory specified by ``out_dir``.
+
+        Parameters
+        ----------
+        input_files : string
+            Path to the input volumes. This path may contain wildcards to
+            process multiple inputs at once.
+        bvalues : string
+            Path to the bvalues files. This path may contain wildcards to use
+            multiple bvalues files at once.
+        bvectors : string
+            Path to the bvalues files. This path may contain wildcards to use
+            multiple bvalues files at once.
+        mask_files : string
+            Path to the input masks. This path may contain wildcards to use
+            multiple masks at once. (default: No mask used)
+        b0_threshold : float, optional
+            Threshold used to find b=0 directions (default 0.0)
+        out_dir : string, optional
+            Output directory (default input file directory)
+        out_tensor : string, optional
+            Name of the tensors volume to be saved (default 'tensors.nii.gz')
+        out_fa : string, optional
+            Name of the fractionnal anisotropy volume to be saved
+            (default 'fa.nii.gz')
+        out_ga : string, optional
+            Name of the geodesic anisotropy volume to be saved
+            (default 'ga.nii.gz')
+        out_rgb : string, optional
+            Name of the color fa volume to be saved (default 'rgb.nii.gz')
+        out_md : string, optional
+            Name of the mean diffusivity volume to be saved
+            (default 'md.nii.gz')
+        out_ad : string, optional
+            Name of the axial diffusivity volume to be saved
+            (default 'ad.nii.gz')
+        out_rd : string, optional
+            Name of the radial diffusivity volume to be saved
+            (default 'rd.nii.gz')
+        out_mode : string, optional
+            Name of the mode volume to be saved (default 'mode.nii.gz')
+        out_evec : string, optional
```

Comment:
Is there any way to *not* save all these volumes? For example, if all you want is FA?